packages/pn-commons/src/utils/__test__/notification.utility.test.ts

```diff
@@ -11,7 +11,7 @@ import {
   TimelineCategory,
   NotificationStatus,
 } from '../../types';
-import { NotificationDeliveryMode, NotificationStatusHistory, ResponseStatus } from '../../types/NotificationDetail';
+import { DigitalDomicileTypeForCourtesyMessageOnly, DigitalWorkflowDetails, INotificationDetailTimeline, NotificationDeliveryMode, NotificationDetail, NotificationDetailRecipient, NotificationStatusHistory, RecipientType, ResponseStatus } from '../../types/NotificationDetail';
```

Comment:
Merge this import with the above one and add missing imports to src/types/index.ts

Reply:
Done.